www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,110 @@
+        <head>
+            <title>Mel Bolen's Gallery</title>
+            <meta charset="utf-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <meta name="description" content="Portfolio for Connor Brezinsky">
+            <meta name="author" content="Connor Brezinsky">
+            <!-- Bootstrap Core CSS -->
+            <link href="css/bootstrap.min.css" rel="stylesheet"><!-- Custom CSS -->
+            <link href="css/stylish-portfolio.css" rel="stylesheet"><!-- Custom Fonts -->
+            <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+            <link href="css/onepage-scroll.css" rel="stylesheet" type="text/css">
+            <link href="css/styles.css" rel="stylesheet">
+			<link href="css/galleryslideshow.css" rel="stylesheet"><!-- For slideshow -->
+            <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
+            <style type="text/css">
+            div.c9 {margin-top:10em;}
+            a.c8 { color: #050504;}
+            div.c7 {margin-top:8em;}
+            p.c6 {font-size:3em;padding-bottom:1em;}
+            p.c5 {font-family: 'Raleway', sans-serif; font-weight: 300;}
+            h1.c4 { padding-right:4em; font-family: 'Raleway', sans-serif; font-weight: 100;}
+            a.c3 { color: #ffffff;}
+            h1.c2 {font-family: 'Raleway', sans-serif; font-weight: 100;}
+            li.c1 {list-style: none; display: inline}
+            </style>
+	</head>
+	<body>
+	</body>
+               <section>
+                    <header id="top" class="header container">
+                        <div class="text-vertical-center">
+                            <h1 class="c2">
+                                Mel Bolen's Gallery
+                            </h1><a class="c3" href="#about"></a>
+                        </div>
+                    </header>
+                </section>
+            <a href="index.html"><img src="./img/returntohomepage.jpg" class="center" alt="no alt text"/></a>
+        <!-- Slideshow container -->
+    <div class="slideshow-container">
+
+      <!-- Full-width images with number and caption text -->
+      <div class="mySlides fade">
+        <div class="numbertext">1 / 5</div>
+        <img src="./img/bol2.jpg" style="width:100%" alt="no alt text">
+        <div class="text">Mel Bolen.</div>
+      </div>
+
+      <div class="mySlides fade">
+        <div class="numbertext">2 / 5</div>
+        <img src="./img/wife.jpg" style="width:100%" alt="no alt text">
+        <div class="text">Mel Bolen's wife.</div>
+      </div>
+
+      <div class="mySlides fade">
+        <div class="numbertext">3 / 5</div>
+        <img src="./img/mbolenmothersday.jpg" style="width:100%" alt="no alt text">
+        <div class="text">Mel Bolen mothersday.</div>
```

Comment:
We need proper spelling. "mothersday" is not proper spelling. It should probably be two words, potentially with capitals.
(The other spelling "error" is "Bolen's", which I will handle.)